internal/2017-10-montreal/pipeline-knowledge-sharing.md

```diff
@@ -12,13 +12,13 @@ Sanitization procedure -- > Raw data is stored forever, but we sanitize it befor
 
 There are some sanitization checks that don't pass due to formatting reasons.
 
-Eventually the data ends up in postgress database. This is not the best database, but it's good enough. As the raw data storage increases, we will need to consider an alternative.
+Eventually the metadata ends up in postgress database. This is not the best database, but it's good enough. As the raw data storage increases, we will need to consider an alternative.
```

Comment:
My understanding is that the new pipeline's database doesn't store data.

Reply:
Depends on the definition of data I guess. Some data does end up in the DB, I am ok with either though.